docs/Users/Tutorials/Testing_Bears.rst

```diff
@@ -0,0 +1,146 @@
+How to use LocalBearTestHelper to test your bears
+=================================================
+
+**coala** has an awesome testing framework so you can write tests for bears
+with ease.
+
+You can use the following to test your bears:
+
+- ``LocalBearTestHelper.check_validity``
+- ``LocalBearTestHelper.check_results``
+- ``verify_local_bears``
+
+Understanding through examples
+------------------------------
+
+**EXAMPLE 1** using ``verify_local_bear``
+
+PyDocStyleBearTest
```

Comment:
yeah that's right, but most of time the bears can already do "some" stuff and the tests aren't that obvious everytime :)
but yeah if you have a minimal example using our bears that's easy and short (`PyDocStyleBear` does not count :P) then sure use that instead, explaining by reallife examples is everytime better than using fictive ones :D :+1: